### DISCUSSION TITLE
Error 12 on Opus (win64)

### DISCUSSION THREAD

**Discussion by asymcon** [savonet/liquidsoap#2199] [User Support]
**Describe the bug**

When trying to open a stream on Icecast server through opus output, this is returned:

`Error 12: Unsupported format!
You must be missing an optional dependency.`
This persists on 2.0.2 and 2.0.1, and likely relates to #973

Would it be possible to check the %opus binding again?

**To Reproduce**
Use icecast opus output with any parameters, 
`output.icecast(%opus(bitrate=64, signal="music")`

**Expected behavior**
a working output as it worked before in Version 1.x.x

**Version details**

    OS: Windows 7 64bit
    Version 2.0.2

**Install method**
from the currently released 2.0.2 zip File

Common issues
n/a

**Comment by toots** [comment]
Hi,

Because it is built statically, the windows binary cannot have both the ffmpeg encoder and all the other encoders relying on the same underlying libraries.

The way to encode and send a `opus` stream on windows is:
```ruby
enc = %ffmpeg(
  format="ogg",
  %audio(
    codec="libopus",
    samplerate=48000,
    b="96k"
  )
)

output.icecast(
  ...,
  enc,
  s
)

**Comment by asymcon** [comment]
Okay, I got confused that my syntax worked before, but that was 4 years ago. 
I'll try to rewrite it based on your example. 
Thanks
